### DISCUSSION TITLE
Would it be a nice feature to have the whole screen option?

### DISCUSSION THREAD

**Discussion by aloisogomes** [fkhadra/react-toastify#755] [Ideas]
I'm wondering about position option to Toast container supports an  option to our toast covers all whole screen. 

Nice idea or it hurts any principle here?

Ps. Should be useful to mobile screens

**Comment by fkhadra** [comment]
Hey, I don't think this is something I would support out of the box given it's outside the scope of what a notification is. That being said, the CSS can easily be overridden to match the behavior your described
